### ISSUE TITLE
[Feature]: Blur hashes für loading pictures

### ISSUE THREAD

**Issue by akoyaxd** [immich-app/immich#564]
### Feature detail

When scrolling fast through the timeline (on web), one can see the Name / ID of the image before it loads. I think it would be great visual improvement to load blur hashes before the thumbnail.

In the Apps, it would look better than the black boxes for fast scrolling as well.

### Platform

Server

**Comment by alextran1502** [comment]
Great idea! I think this also depends on the browser that the alt value is shown when there is no picture loaded yet. I assume you use Firefox?

**Comment by akoyaxd** [comment]
Yes, I use firefox. Didn't try Chrome before, but I can confirm, that on chrome the string is not shown.

**Comment by alextran1502** [comment]
I love the blur hash ideal, though; I honestly didn't know what it is called. Gonna implement this at some points

**Comment by zackpollard** [comment]
This was recommended on one of the reddit posts, looks like it has implementations for a bunch of languages 😄 https://github.com/woltapp/blurhash

**Comment by JVT038** [comment]
@bo0tzz can I be assigned to this one? I'd like to try learn a bit of Sveltekit :)

**Comment by bt90** [comment]
To add a screenshot of the current state:

![grafik](https://user-images.githubusercontent.com/1926175/209689888-57efd7ac-30a5-4aa7-869b-41104af0fca9.png)

**Comment by bt90** [comment]
As an interim solution, it would be nice to hide the image UUIDs.
